pylcurve/utils.py

```diff
@@ -1,13 +1,74 @@
 import numpy as np
-
-from astropy import units as u
+import astropy.units as u
+from scipy.integrate import quad
 from astropy.constants import G
+from astropy.modeling.blackbody import blackbody_lambda, blackbody_nu
+from .blackbody import bb_interpolator
+from .limbdark import ld_interpolator
+from .massradius import mr_interpolator
+from .rochedistortion import roche_interpolator
+from dust_extinction.parameter_averages import F20
```

Comment:
dust_extinction? Is this an external package?

Reply:
Yeah it's an external package